paddle/fluid/operators/math/math_function_test.cu

```diff
@@ -344,6 +359,11 @@ TEST(math_function, gemm_trans_cublas_fp16) {
   using namespace paddle::framework;
   using namespace paddle::platform;
 
+  // fp16 GEMM in cublas requires GPU compute capability >= 53
+  if (GetCUDAComputeCapability(0) < 53) {
```

Comment:
I am not sure if this `GetCUDAComputeCapability(0) < 53` code should be moved into math_function.cu?

Reply:
Adding processing code for compute capability < 53 case could be a future TODO item. 
Before that is done, I think we need code like this to skip the fp16 test if necessary, otherwise one of the CI machine with old GPUs will fail the test.
I have also added `PADDLE_ENFORCE_GE(compute_cap, 53)` in `math_function.cu` to give a more specific error message for users.